aten/src/ATen/native/mps/MPSGraph+CoolMethods.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#import <MetalPerformanceShadersGraph/MPSGraph.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+// ScatterGather.mm and TriangularOps.mm call to those
+// But they can not be found in https://developer.apple.com/documentation/metalperformanceshadersgraph/mpsgraph?language=objc
+// Nor in MacOSX12.3.sdk headers
+// TODO: Do not call to undocumented methods
+
+@interface MPSGraph(CoolMethods)
+-(MPSGraphTensor *) gatherAlongAxisWithUpdatesTensor:(MPSGraphTensor *) updateTensor
+                                       indicesTensor:(MPSGraphTensor *) indicesTensor
+                                                axis:(NSInteger)dim
+                                                name:(NSString * _Nullable)name;
+
+-(MPSGraphTensor *) getCoordinateValueWithShapeTensor:(MPSGraphTensor *) shapeTensor
```

Comment:
@malfet, can you please replace these methods with the Public SDK ones, also please remove this file. We won't need CoolMethods 😄 .
```
-(MPSGraphTensor *) coordinateAlongAxisTensor:(MPSGraphTensor *) axisTensor
                              withShapeTensor:(MPSGraphTensor *) shapeTensor
                                         name:(NSString * _Nullable) name
```
Note the axisTensor and shapeTensor are swapped.